util/stream_test.go

```diff
@@ -0,0 +1,166 @@
+package util
+
+import (
+	"bytes"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSmallMessageError(t *testing.T) {
+	smallOpenFlowErr := newSmallMessageError(5)
+	var smallOFErr *smallMessageError
+	assert.True(t, errors.As(smallOpenFlowErr, &smallOFErr))
+}
+
+type fakeWorker struct {
+	InputCh chan *bytes.Buffer
+}
+
+const (
+	vendor                   = uint32(0x00002320)
+	experimenterForPacketIn2 = uint32(30)
+)
+
+func TestInbound_LargePacket_TableDriven(t *testing.T) {
+	tests := []struct {
+		name             string
+		packetLength     uint16
+		headerLength     uint16
+		vendorType       uint32
+		experimentorType uint32
+	}{
+		{"SmallPacket", 64, 84, vendor, experimenterForPacketIn2},
+		{"MediumPacket", 1500, 1520, vendor, experimenterForPacketIn2},
+		{"LargePacket", 65530, 14, vendor, AntreaLargePacket},
```

Comment:
that seems to imply you are using the same vendor id as nicira

Reply:
Yes, I use `AntreaLargePacket` as a value of field `ExperimenterType`, the role as "PacketIn2" (value=30) but different value.